packages/@glimmer/compiler/lib/passes/1-normalization/keywords/impl.ts

```diff
@@ -8,8 +8,8 @@ import type { NormalizationState } from '../context';
 import { Err } from '../../../shared/result';
 
 export interface KeywordDelegate<Match extends KeywordMatch, V, Out> {
-  assert(options: Match, state: NormalizationState): Result<V>;
-  translate(options: { node: Match; state: NormalizationState }, param: V): Result<Out>;
+  assert: (options: Match, state: NormalizationState) => Result<V>;
```

Comment:
Since `assert` and `translate` are extracted, this documents that they are arrows without a `this` binding.